### ISSUE TITLE
Decoder training

### ISSUE THREAD

**Issue by CT001002** [luozhenjie1997/CPL-Diff#2]
Thank you so much for your work! In your paper, "The parameters of the encoding part will not be adjusted during the entire training process, while the decoding part will use the collected polypeptide sequences for fine-tuning". So is the decoding part trained with the fixed encoding part together? and for each label the encoder-decoder is trained by each label dataset? Would you kindly share the detailed decoding training code? That would be very helpful.
